docs/app-store-connect-api.md

```diff
@@ -92,6 +92,19 @@ end
 Keys and values that can be used in hash in `api_key` parameter in actions (e.g. `upload_to_testflight`) and tools (e.g. `cert`) are described in _fastlane_ API Key JSON file format below.
 The only difference when using hash is that you could use `filepath` instead of `key`.
 
+Below is an example of API key being used with CLI:
+
+```no-highlight
+$ bundle exec fastlane match appstore \
+    --api_key "
+        {
+            \"filepath\": \"/Users/bartek/Downloads/AuthKey_TJP2GMDZCI.p8\",
+            \"key_id\": \"TJP2GMDZCI\",
+            \"issuer_id\": \"57246542-96fe-1a63-e053-0824d011072a\"
```

Comment:
To double-check, are they valid values? 😄 We can't access the API without the AuthKey file but make sure nothing bad will happen by using dummy values or invalidating the above key_id/issuer_id.

Reply:
These values are not real :)